dashboards-reports/public/components/report_definitions/delivery/delivery.tsx

```diff
@@ -110,53 +119,92 @@ export function ReportDelivery(props: ReportDeliveryProps) {
   }
 
   const defaultCreateDeliveryParams = () => {
+    includeDelivery = false;
     reportDefinitionRequest.delivery = {
       configIds: [],
-      title: '',
-      textDescription: '',
+      title: `\u2014`, // default values before any Notifications settings are configured
+      textDescription: `\u2014`,
       htmlDescription: ''
     };
   };
 
-  const sendTestNotificationsMessage = () => {
-    // implement send message
+  const sendTestNotificationsMessage = async () => {
     // on success, set test message confirmation message
-    handleTestMessageConfirmation(testMessageConfirmationMessage)
+    // for each config ID in the current channels list
+    for (let i = 0; i < selectedChannels.length; ++i) {
+      try {
+        const eventId = await httpClientProps
+          .get(`../api/reporting_notifications/test_message/${selectedChannels[i].id}`, 
+          {
+            query: {
+              feature: 'reports'
+            }
+          })
+          .then((response) => response.event_id);
+          
+        await httpClientProps
+          .get(`../api/reporting_notifications/get_event/${eventId}`)
+          .then((response) => {
+            if (response.event_list.length != 1 && response.event_list[0].event_id != eventId) {
+              const error = new Error('Failed to send the test message.');
+              throw error;
+            }
+            else {
+              handleTestMessageConfirmation(testMessageConfirmationMessage);
+            }
+          });
+      } catch (error) {
+        // error message
+        handleTestMessageConfirmation(testMessageFailureMessage);
```

Comment:
Should the test message be more verbose and contains info about which channel is failing for a test message

Reply:
User needs to know what goes wrong, and get enough info to resolve the issue from his end.

Reply:
Fixed, now test message failure message provides list of channels that failed to send test message